Automattic/jetpack
# Make VideoPress a hybrid product in My Jetpack

Fixes #25941

#### Changes proposed in this Pull Request:

* Make VideoPress a hybrid product in My Jetpack

#### Other information:


#### Jetpack product discussion


pe4Cmx-fj-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






First, on the monorepo:

* Connect the site at a site and user level
* Activate Jetpack Protect
* Now deactivate both Jetpack and VideoPress
* Go to My Jetpack
* Activate VideoPress - make sure the card turns into "Active" state
* Click on the Manage button and make sure it points to the VideoPress admin UI
* Make sure the VideoPress plugin was activated (and not Jetpack)
* Repeat the test above starting from a completely disconnected site - after activating VideoPress you should see a message asking you to connect your user
* Deactivate and "uninstall" VideoPress. On the terminal:
```
jetpack docker wp plugin deactivate videopress
jetpack docker unlink-plugin videopress
```
* Go to My Jetpack
* Activate VideoPress
* make sure the card turns into "Active" state
* Click Manage
* make sure you go to Jetpack Settings page - with "videopress" filtered.
* Make sure Jetpack plugin was activated
```
jetpack docker link-plugin videopress
```
* Run the command above to "reinstall" videopress

Now, on a JN site:
* launch a JN site on this branch only with Protect installed
* Connect
* Go to My Jetpack
* Activate VideoPress, go through the connection flow. Make sure it all works.
